components/app/leaderboard/FloatingVoteAction.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div
+    class="
+      fixed
+      w-full
+      flex
+      justify-end
+      lg:top-1/2
+      bottom-5
+      pointer-events-none
+    "
+  >
+    <app-link :to="voteUrl" :data-analytics="dataAnalytics">
+      <div
+        class="bg-white rounded-l shadow p-4 pointer-events-auto text-center"
+      >
+        <div
```

Comment:
question(non-blocking); can we loose this div and add its classes to the app-icon component?

Reply:
This div contains also the 'Vote' title so we cannot move this inside app-icon itself.

Reply:
this div only wrappes the app-icon, the parent div contains the leaderboard.vote title. But i guess it is not that important since we have this component only once inside the dom.